pkg/diff/diff_test.go

```diff
@@ -214,7 +214,7 @@ func testDataEqual(dbConn *sql.DB, sourceTables []string, targetTable string, ha
 		_, err = dbConn.Exec(sql)
 		c.Assert(err, IsNil)
 	}
-	structEqual, dataEqual, err = tableDiff.Equal(context.Background(), writeSqls)
+	structEqual, dataEqual, err = tableDiff.Equal(context.Background(), nil)
```

Comment:
why change `writeSqls` to nil here.
if checked data is equal, there is no side effect. But if checked data is not equal (of course the inequality is caused by bug, but during development it may happen), the test will panic.

Reply:
done